### DISCUSSION TITLE
Separate indexer deployment doesn't work when deploying via blockscout-stack helm chart

### DISCUSSION THREAD

**Discussion by scheibinger** [blockscout/blockscout#11087] [Support] [answered]
I tried deploying the blockscout-stack chart and enabling the separate indexer deployment via helm config:

```
  separateApi:
    enabled: true
```

Unfortunately that doesn't seem to work as the `docker.io/blockscout/blockscout:latest-indexer` image with `latest-indexer` tag is not available.
I'm also not able to override the image tag via helm values as the `-indexer` suffix is hardcoded [here](https://github.com/blockscout/helm-charts/commit/62d7533c9c9aeb1f12d2656c1863bfe9ec372f62#diff-01358a9c4e9a9c1bce586d40dfba673357c9e98a239d98b31c4d437230fa4d4cR240)

The separate indexer feature was introduced in this [commit](https://github.com/blockscout/helm-charts/commit/62d7533c9c9aeb1f12d2656c1863bfe9ec372f62#diff-30d8aec23edb627f7979d941a3ee735f5fdfacf433f481f15d8d1da4d8b03044R5) and I was hoping it could help to solve the stability issues I'm facing when deploying to EKS kubernetes cluster.

**Comment by scheibinger** [comment]
I was able to fix that by removing the indexes in this PR: https://github.com/blockscout/helm-charts/pull/43

**Reply by GimluCom** [reply, marked as answer]
You may also achieve the same result by replacing ![](https://img.shields.io/badge/tag%3A%20%22latest%22-black?style=flat-square&color=rgba%280%2C0%2C0%2C0.5%29) with ![](https://img.shields.io/badge/dynamic/json?url=https%3A%2F%2Fjqp.vercel.app%2Fapi%2Fv0%3Furl%3Dhttps%253A%252F%252Fjqp.vercel.app%252Fapi%252Fv0%253Furl%253Dhttps%253A%252F%252Fapi.github.com%252Frepos%252Fblockscout%252Fblockscout%252Freleases%252Flatest%2526jq%253D.tag_name%257Cltrimstr(%252522v%252522)&query=%24&prefix=tag%3A%20%22&suffix=%22&style=flat-square&label=%20&color=rgba%280%2C0%2C0%2C0.5%29&cacheSeconds=172800) in [values.yaml](https://github.com/blockscout/helm-charts/blob/4af3db0104bbb4b30c3d2834b2e04316b784fc6e/charts/blockscout-stack/values.yaml#L79)

**Reply by scheibinger** [reply]
thanks a lot, that actually worked. Previously I tried pulling `v6.8.1` tag, but that wouldn't work have suffixes. If I use the tag format that you mentioned, it works 🙏
